app/resources/applications/ApplicationsController.jl

```diff
@@ -17,6 +17,8 @@ using Dates
 const appsthreads = Dict()
 const apphost = "http://127.0.0.1"
 
+const PORTS_RANGE = 9101:9110
```

Comment:
Please move this into the env files, so we can have different range in dev (when running locally)